roles/nginxplus/tasks/conf/debug-output.yml

```diff
@@ -1,9 +1,9 @@
 ---
-- name: "Setup: Register NGINX configuration"
-  command: nginx -T
+- name: nginxplus | Register NGINX configuration
+  ansible.builtin.command: nginx -T
   changed_when: false
   register: nginx_configuration
 
-- name: "Setup: Print NGINX configuration"
-  debug:
+- name: nginxplus | Print NGINX configuration
+  ansible.builtin.debug:
```

Comment:
Does this run in testing? If so, does it fail idempotency? The documentation isn't clear, but my bet is that the `debug` module always returns `changed`. You've already got `changed_when: false` on the prior task . . .